rotkehlchen/tests/unit/test_accounting.py

```diff
@@ -969,6 +969,68 @@ def test_fees_in_received_asset(accountant):
     assert FVal(report['total_profit_loss']) == FVal(14.13870)
 
 
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+@pytest.mark.parametrize('dont_mock_price_for', [[A_KFEE]])
+def test_kfee_price_in_accounting(accountant):
+    """
+    Test that KFEEs are corrently handled during the accounting
+    """
+    history = [
+        {
+            'timestamp': 1609537953,
+            'base_asset': 'ETH',
+            'quote_asset': A_USDT.identifier,
+            'trade_type': 'sell',
+            'rate': 1000,
+            'fee': '30',
+            'fee_currency': A_KFEE.identifier,
+            'amount': 0.02,
+            'location': 'binance',
+        },
+    ]
+    ledger_actions_list = [
+        LedgerAction(
+            identifier=0,
+            timestamp=Timestamp(1539713238),
+            action_type=LedgerActionType.INCOME,
+            location=Location.BINANCE,
+            amount=FVal(1),
+            asset=A_ETH,
+            rate=None,
+            rate_asset=None,
+            link=None,
+            notes='',
+        ), LedgerAction(
+            identifier=0,
+            timestamp=Timestamp(1539713238),
+            action_type=LedgerActionType.INCOME,
+            location=Location.BINANCE,
+            amount=FVal(1000),
+            asset=A_KFEE,
+            rate=None,
+            rate_asset=None,
+            link=None,
+            notes='',
+        ),
+    ]
+    report, _ = accounting_history_process(
+        accountant,
+        start_ts=1539713238,
+        end_ts=1624395187,
+        history_list=history,
+        ledger_actions_list=ledger_actions_list,
+    )
+    warnings = accountant.msg_aggregator.consume_warnings()
+    assert len(warnings) == 0
+    errors = accountant.msg_aggregator.consume_errors()
+    assert len(errors) == 0
+    # The ledger actions income doesn't count for the income as the 1
```

Comment:
There is no sell. Is this comment copy pasta?

Reply:
There is one sell (the one that has the KFEE) and two ledger actions as income

Reply:
Hmmm okay